.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
Little resistant to adding personal editor settings to the repo, even mine.

Reply:
You know, there are different levels of "editor settings" (convention): "team" (it can have several projects), "projects" (where we're), "personal".

And I respect any level of preferences, but was trying to introduce project-wide convention. To keep project's source code in one style (not to mix space-indentation with tabs, etc.)

If you're against it — I can remove. But in my practice it helps, not to specific maintainers or contributors, but to a project, and all its members (how to write code for the specific project despite personal preferences).

Reply:
Like an additional example:

I have personal taste in tab-indentation. Even in Ruby. But I don't enforce it. And I have a code editor plugin, which toggles my settings (indentation, line length / wrapping, encoding, etc.) to the specific open project.

I wish similar to everyone. You can write as you want, but in a single project we're all should stick together, with chosen rules.

Reply:
@AlexWayfer fair points; how about this, fine with leaving it if we bump the `.editorconfig` / rubocop line length to the current rubocop default of 120, everything else seems fairly reasonable for what I'd expect from a ruby ide.

Reply:
> @AlexWayfer fair points; how about this, fine with leaving it if we bump the `.editorconfig` / rubocop line length to the current rubocop default of 120, everything else seems fairly reasonable for what I'd expect from a ruby ide.

What is "ruby ide"? I've been using Atom Editor, now I use Pulsar Editor. I can use VS Code — whatever. 

You can open EditorConfig's site: https://editorconfig.org/#pre-installed

Some editors work with it "out-of-box", some requires (simple) plugins. Even RubyMine, Sublime, etc.

Again: if I have editor's setting for tab-indentation — I'd have problems with this project with 2-spaces indentation. Which I'm not against, this config is just a kind of automatization for project-to-project work.

The same for end-of-line chars between UNIX and Windows users on the same projects, etc. Nobody against Windows users or their end-line chars, we just should keep the same chars along the project.